website/views/main/filters.jade

```diff
@@ -21,7 +21,7 @@
         li.filters-edit(ng-show='_editing')
           form.hrpg-input-group
             input(type='text', ng-model='_newTag', placeholder=env.t('newTag'))
```

Comment:
Totally correct, I made some poor git decisions and somehow lost those lines by the time I pushed my changes.